### PR TITLE
Update to dev process

### PR DESCRIPTION
**Pull request by maayanf24**
Remove special sprint for epic design.
Remove exception process for epics.

### CODE REVIEW

File: src/content/development/contribution-guide/_index.en.md [hunk 1]
```diff
@@ -43,6 +43,7 @@ or as a general [Enhancement Request issue].
 
 Submariner's release is automated to a great extent. [The release process documentation] explains the details of a release. The release
 automation is maintained in the [releases repository]. It also hosts `subctl` binaries for all released versions.
+During the release process there should be a code-freeze.
```
Comment (skitt) [inline src/content/development/contribution-guide/_index.en.md]: Let’s be categorical and more explicit:

```suggestion
Code is frozen during releases: no PRs unrelated to the release can be merged to the branch being released.
```

File: src/content/development/contribution-guide/_index.en.md [hunk 2]
```diff
@@ -95,7 +96,7 @@ Each cycle will result in either a minor version bump or a major version bump in
 ### Sprints
 
 Sprints are 3 week periods which encapsulate work on the release.
-Most sprints focus on active development of the current version, while some focus on additional aspects such as design and stabilization.
+Most sprints focus on active development of the current version, while the last one focus on stabilization.
```
Comment (skitt) [inline src/content/development/contribution-guide/_index.en.md]:
```suggestion
Most sprints focus on active development of the current version, while the last one focuses on stabilization.
```

File: src/content/development/contribution-guide/_index.en.md [hunk 3]
```diff
@@ -210,7 +205,7 @@ Any [unplanned work](#unplanned-work) should follow the defined guidelines.
 Each sprint will end with a release to allow the community to test new features and fixed bugs.
 In total, three milestone sprints are planned:
 
-* Two milestones ending with releases of milestone `m1` and milestone `m2`.
+* Three milestones ending with releases of milestone `m1`, `m2` and `m3`.
```
Comment (skitt) [inline src/content/development/contribution-guide/_index.en.md]:
```suggestion
* Three sprints ending with the release of each milestone `m1`, `m2`, and `m3`.
```

**Review by skitt — CHANGES_REQUESTED** [review]

File: src/content/development/contribution-guide/_index.en.md [hunk 4]
```diff
@@ -336,7 +313,6 @@ Should a bug be identified during a test day, it should be labeled with an appro
 [test day template]: https://docs.google.com/spreadsheets/d/14wNJAbR2CC3wWdHtxnvnNlft8kSWp-SIFMWJ-hRISao
 [Submariner project]: https://github.com/submariner-io
 [triages tasks and bugs as part of a weekly meeting]: https://docs.google.com/document/d/1ICvOBOUpIEI45Yqcjf26Z1e5mZNzYsEN-H7_3xx54jI
-[Project Members]: https://submariner.io/community/contributor-roles/#member
```
Comment (vthapar) [inline src/content/development/contribution-guide/_index.en.md:339]: Why this change? Doesn't seem related.

Comment (maayanf24) [reply src/content/development/contribution-guide/_index.en.md:339]: The link was referenced from a line which was removed in this PR.
Line 294, as part of the "Exception Process"

**Review by skitt — APPROVED** [review]

**Review by Jaanki — APPROVED** [review]

**Review by vthapar — APPROVED** [review]
